dist/scripts/main.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const fullNameRegExp = new RegExp(
+  "^([A-Z][a-z]*'?[a-z]*[-]?[A-Z]?[a-z]*('[a-z]*)?\\s){1,2}"
+  + "[A-Z][a-z]*'?[a-z]*[-]?[A-Z]?[a-z]*('[a-z]*)?$"
+);
+const phoneNumberRegExp = /^(((\+?3)?\s?8\s?)?0)?(\d{2,4}(\s|-)?){1,4}$/;
+const citiesByRegionsObject = {
+  'Kyiv': [],
+  'Center':
+    ['Cherkasy', 'Dnipro', 'Kropyvnytskyi', 'Poltava', 'Vinnytsia', 'Zhytomyr'],
+  'North': ['Chernihiv', 'Sumy'],
+  'East': ['Donetsk', 'Kharkiv', 'Luhansk'],
+  'South': ['Kherson', 'Mykolaiv', 'Odesa', 'Zaporizhzhia'],
+  'West':
+    ['Chernivtsi', 'Ivano-Frankivsk', 'Khmelnytskyi', 'Lutsk', 'Lviv', 'Rivne',
+      'Ternopil', 'Uzhhorod'],
+};
+
+function main() {
+  const registrationForm = document.forms.registrationForm;
+  const fullName = document.registrationForm.fullName;
+  const needMore = document.registrationForm.needMore;
+  const phoneNumber = document.registrationForm.phoneNumber;
+  const region = document.registrationForm.region;
+  const city = document.registrationForm.city;
+  const submitButton = registrationForm.querySelector('input[type="submit"]');
+
+  const initBlankInSelect = function(selectElement) {
+    const blankOption = document.createElement('option');
+
+    selectElement.options.length = 0;
+    blankOption.value = 'blank';
+    blankOption.text = '';
+    selectElement.add(blankOption, null);
+  };
+
+  const validateAll = function() {
+    let isValid;
+    const backgroundRed = function(someInput) {
+      someInput.classList.remove('inputs-column_valid');
+      someInput.classList.add('inputs-column_invalid');
+    };
+    const backgroundGreen = function(someInput) {
+      someInput.classList.remove('inputs-column_invalid');
+      someInput.classList.add('inputs-column_valid');
+    };
+    const emptied = function(someInput) {
+      someInput.classList
+        .remove('inputs-column_invalid', 'inputs-column_invalid');
+    };
+
+    isValid = fullNameRegExp.test(fullName.value.trim());
+    if (fullName.value !== '') {
+      if (isValid) {
+        backgroundGreen(fullName);
+      } else {
+        backgroundRed(fullName);
+      }
+    } else {
+      emptied(fullName);
+    }
+    if (!needMore.checked) {
+      const nextResult = phoneNumberRegExp.test(phoneNumber.value.trim());
+      if (phoneNumber.value !== '') {
+        if (nextResult) {
+          backgroundGreen(phoneNumber);
+        } else {
+          backgroundRed(phoneNumber);
+        }
+      } else {
+        emptied(phoneNumber);
+      }
+      isValid &= nextResult;
```

Comment:
Seniors and tech leads in my company disable bit operators by eslint because it is too hard to use them😂😂😂

It is ok, feel free to use all the tools which you have if you knew what you do